### PR TITLE
Updated 'Compilation' section in README.md

### PR DESCRIPTION
**Pull request by matusmarhefka**
I mainly added packages required for `make check` and the requirement to run sendmail service.

### CODE REVIEW

**Comment by dahaic** [comment]
That result is appealing! CI test this again please

**Comment by dahaic** [comment]
@openscap-ci test this again please

**Comment by dahaic** [comment]
That result is appalling! CI test this again please.

**Comment by dahaic** [comment]
Jenkins test this please.

File: README.md [hunk 1]
```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
...
+systemctl start sendmail.service
+```
+Now you can execute the following command to run library self-checks:
 ```
 make check
 ```
+Note: If you want to run `make distcheck` you will also need to install
+`rubygems` package and install `asciidoctor`:
+```
+gem install asciidoctor
```
Comment (jan-cerny) [inline README.md]: There is a package `rubygem-asciidoctor` on my Fedora, so you don't have to run `gem`, you can run only dnf.

File: README.md [hunk 2]
```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
...
+autoconf automake libtool gcc
+```
+
+2) a) Get a release tarball and run ./autogen.sh:
 ```
 # replace ${version} with the desired version
 wget https://github.com/OpenSCAP/openscap/releases/download/${version}/openscap-${version}.tar.gz
 tar -xzpf openscap-${version}.tar.gz
 cd openscap-${version}
+./autogen.sh
```
Comment (jan-cerny) [inline README.md]: Is autogen.sh really needed on download tarball? I thought it was not mentioned here by purpose.

File: README.md [hunk 3]
```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
...
+`rubygems` package and install `asciidoctor`:
+```
+gem install asciidoctor
+```
 
-4) Run the installation procedure by executing the following command:
+5) Run the installation procedure by executing the following command:
 ```
 make install
 ```
```
Comment (jan-cerny) [inline README.md]: Awesome

File: README.md [hunk 4]
```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
 
 ## Compilation
 
-Choose *1a* or *1b* depending on whether you want sources from a release tarball or the git repository.
+Choose *2a* or *2b* depending on whether you want sources from a release tarball or the git repository.
 
-1a) Get a release tarball
+1) The following packages are required to be installed on your system:
+```
+autoconf automake libtool gcc
```
Comment (mpreisler) [inline README.md]: they are not required if you use release tarball

Comment (matusmarhefka) [inline README.md]: Thanks for the review, I fixed that in dab3f4f

File: README.md (hunk 2 above)
Comment (matusmarhefka) [inline README.md]: true

File: README.md (hunk 1 above)
Comment (matusmarhefka) [inline README.md]: I fixed this in dab3f4f - only fedora has `rubygem-asciidoctor` package, on RHEL and/or other distros without this package you will need to install `rubygems` package.

**Comment by jan-cerny** [comment]
I have read it again and it looks ok. Thank you.

**Pull request merged**